lib/body.js

```diff
@@ -0,0 +1,120 @@
+import {EventEmitter2} from 'eventemitter2';
+import {Message} from 'roslib';
+
+class Body extends EventEmitter2 {
+  constructor(robot) {
+    super();
+
+    const ros = robot.ros;
+
+    // Define the action
+    this.action = new ROSLIB.ActionClient({
+      ros: ros,
+      serverName: 'body/joint_trajectory_action',
+      actionName: 'control_msgs/FollowJointTrajectoryAction',
+      timeout: 10,
+    });
+
+    // Get the skills parameters from the parameter server to extract the
+    // possible motions.
+    // ToDo: get rid of hardcoded "amigo"
```

Comment:
We'd have to check this I guess: does it still work if we remove `/amigo/`. Rosbridge fixes the namespacing?